libsignal-service/src/websocket/sender.rs

```diff
@@ -3,11 +3,11 @@ use crate::sender::{OutgoingPushMessages, SendMessageResponse};
 use super::*;
 
 impl SignalWebSocket {
-    pub async fn send_messages<'a>(
+    pub async fn send_messages(
         &mut self,
-        messages: OutgoingPushMessages<'a>,
+        messages: OutgoingPushMessages,
     ) -> Result<SendMessageResponse, ServiceError> {
-        let path = format!("/v1/messages/{}", messages.destination);
+        let path = format!("/v1/messages/{}", messages.recipient);
```

Comment:
Same here, try not to depend on the Display implementation of a struct that should be an opaque type. Explicitely request the uuid here.